packages/code/src/Code.tsx

```diff
@@ -263,7 +248,7 @@ function Code({
   children = '',
   className,
   multiline = true,
-  language = Language.Auto,
+  language,
```

Comment:
Does this not need a default anymore?

Reply:
I think we're making it required now.

We could default to None but that's a poor user experience.

I don't have a strong opinion on this, I see arguments on both sides.

Reply:
Plugins aren't supported with auto highlighting, so we're dropping support for it for the time being. Usually folks will want to specify a language anyways, so removing auto shouldn't be the biggest deal.

I think that aside from very isolated cases, syntax highlighting will be desired, so having them specify 'none' in those edge cases shouldn't be a huge deal.